### DISCUSSION TITLE
Planning to use Haraka for transactional and newsletters - help me understand

### DISCUSSION THREAD

**Discussion by jesperordrup** [haraka/Haraka#3145] [Q&A]
Hi all,

I really would like to use Haraka as a smtp server for replaying our outgoing transactional emails and for newsletters. Smtp transactional is not my worry but using it to send out newsletters to 40.000 -100.000 recepients on a daily basis is what worries me. 

**Newsletters / mass mail**
Is anyone doing the same?

Can you help clarifying how much work is it to run a smtp server on a daily basis? What is taken up your time?

- keep clear of blacklistning?
- manual stuff?

**Any Haraka tips**
Do you have any fantastic plugin suggestions or other tips. Please share.

**Performance**
Im not so much worried about performance. I assume that Haraka is up for it with the right sized container(s). 

**Statistics**
Im considering adding a custom plugin for modifying email content for tracking. Any suggestions?

Thank you!  
Jesper

**Comment by msimerson** [comment]
> but using it to send out newsletters to 40.000 -100.000 recipients on a daily basis is what worries me

What exactly are you worried about? Unless your newsletters are multi-megabyte monsters and your bandwidth is sorely limited, any MTA can handle this with 1.5 arms tied behind it's back.

**Reply by jesperordrup** [reply]
Thanks @msimerson - sorry for the delay here.

Im actually only worried about keeping the ip clean (not in blacklists) so currently we have a setup with a professional smtp provider which works well.

But what I need is some way to modify an email between our internal smtp clients and sending using our existing smtp provider.

Im thinking about setting up Haraka as an "in between" smtp server. I dont know the term if there is any (help?). The Haraka server would receive emails from internal systems, work some haraka plugin magic (modify html body) and then pass on all emails to our smtp provider for the "real" sendout. 

Makes sense?
